Sources/tuist/tuist.docc/Articles/Tuist/Users/synthesized-files.md

```diff
@@ -0,0 +1,97 @@
+# Synthesized files
+
+Learn how Tuist synthesizes code and resources at generation-time
+
+## Overview
+
+### Target resources
+
+Xcode projects support adding resources to targets. However, they present teams with a few challenges, specially when working with a modular project where sources and resources are often moved around:
+
+- **Inconsistent runtime access**: Where the resources end up in the final product and how you access them depends on the target product. For example, if your target represents an application, the resources are copied to the application bundle. This leads to code to access the resources that makes assumptions on the bundle structure, which is not ideal because it makes the code harder to reason about and the resources to move around.
```

Comment:
```suggestion
- **Inconsistent runtime access**: Where the resources end up in the final product and how you access them depends on the target product. For example, if your target represents an application, the resources are copied to the application bundle. This leads to code accessing the resources that makes assumptions on the bundle structure, which is not ideal because it makes the code harder to reason about and the resources to move around.
```